--- Wav35SkinApplet.lua.orig	2015-09-25 15:04:12.000000000 -0400
+++ Wav35SkinApplet.lua	2017-09-22 16:58:39.000000000 -0400
@@ -275,7 +275,7 @@
 
 
 	local blackBackground   = Tile:fillColor(0x000000ff)
-	local nocturneWallpaper = _loadImageTile(self, "applets/SetupWallpaper/wallpaper/fab4_nocturne.png")
+	-- local nocturneWallpaper = _loadImageTile(self, "applets/SetupWallpaper/wallpaper/fab4_nocturne.png")
 
 	--FIXME, _r asset here doesn't work...it's supposed to have a fadeout effect and it doesn't appear on screen
 	local fiveItemBox             = _loadHTile(self, {
@@ -3214,7 +3214,7 @@
 	-- Visualizer: Container with titlebar, progressbar and controls.
 	--  The space between title and controls is used for the visualizer.
 	s.nowplaying_visualizer_common = _uses(s.nowplaying, {
-		bgImg = nocturneWallpaper,
+		bgImg = blackBackground,
 
 		npartistgroup = { hidden = 1 },
 		npalbumgroup = { hidden = 1 },
